homeassistant/components/modbus.py

```diff
@@ -33,10 +34,12 @@
 _LOGGER = logging.getLogger(__name__)
 
 SERVICE_WRITE_REGISTER = "write_register"
+SERVICE_WRITE_REGISTERS = "write_registers"
 
 ATTR_ADDRESS = "address"
 ATTR_UNIT = "unit"
 ATTR_VALUE = "value"
+ATTR_VALUES = "values"
```

Comment:
It should work after https://github.com/home-assistant/home-assistant/pull/3255 is merged.